src/Controller/ParticipantDetailsController.php

```diff
@@ -172,6 +175,23 @@ public function participantDetailsAction(
         // Incentive Form
         $incentiveForm = $this->createForm(IncentiveType::class, null, ['disabled' => $this->isReadOnly()]);
 
+        // Id Verification Form
+        $idVerificationForm = $this->createForm(IdVerificationType::class, null, ['disabled' => $this->isReadOnly()]);
+        $idVerificationForm->handleRequest($request);
+        if ($idVerificationForm->isSubmitted()) {
+            if ($idVerificationForm->isValid()) {
+                if ($idVerificationService->createIdVerification($id, $idVerificationForm->getData())) {
+                    $this->addFlash('success', 'ID Verification Saved');
+                } else {
+                    $this->addFlash('error', 'Error saving id verification . Please try again');
+                }
+            } else {
+                $this->addFlash('error', 'Invalid form');
+            }
+            return $this->redirectToRoute("participant", ['id' => $id]);
```

Comment:
Not sure if there's something different with the multiple forms on this page, but wouldn't we only want to redirect if the form submission was successful? Right now, it would just display a generic "invalid form" and the form would be cleared, instead of displaying the form with their previous submission and inline errors.